.github/workflows/build.yml

```diff
@@ -349,8 +349,35 @@ jobs:
       - name: Install rust toolchain
         uses: dtolnay/rust-toolchain@stable
       - name: Install cross
-        uses: taiki-e/install-action@cross
-      - run: cross build -p rustls --locked --target i686-unknown-linux-gnu
+        uses: taiki-e/install-action@v2
+        with:
+          tool: cross
+      - run: cross build --package rustls --locked --target i686-unknown-linux-gnu
+      - run: cross test --package rustls --target i686-unknown-linux-gnu
+
+  cross-test-targets:
+    name: cross build-test - multiple targets
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        target:
+          - aarch64-linux-android
```

Comment:
I am not so convinced we should keep this `aarch64-linux-android` target, as this CPU type should already be covered by testing on normal macOS silicon. I would favor removing this in favor of adding some more exotic targets & other embedded CPU/OS combinations.